class-amp-enhancer-form-sanitizer.php

```diff
@@ -219,7 +220,7 @@ public function ensure_response_message_elements( $form ) {
 				$form->appendChild( $input_span );
 		    }
 
-		$templates = $this->dom->xpath->query( Document::XPATH_MUSTACHE_TEMPLATE_ELEMENTS_QUERY, $form );
+		$templates = $this->dom->xpath->query( MustacheScriptTemplates::XPATH_MUSTACHE_TEMPLATE_ELEMENTS_QUERY, $form );

```

Comment:
In order to be compatible with AMP v2.1 as well, I think we'll have to resort to using a string instead of using a class constant.



cf. https://github.com/ampproject/amp-toolbox-php/pull/434
```suggestion
		$templates = $this->dom->xpath->query( './/self::template[ @type = "amp-mustache" ]|//self::script[ @type = "text/plain" and @template = "amp-mustache" ]', $form );
```